### PR TITLE
220718 update bullseye

### PR DESCRIPTION
**Pull request by nalssing**
## Changes
- Update base image to bullseye.
- It supports ffmpeg 4.3
- Required to update bepro-python-api base image

### CODE REVIEW

File: python_with_ffmpeg/Dockerfile [hunk 1]
```diff
@@ -16,19 +16,20 @@ RUN curl -sSLO https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google
...
-       git=1:2.20.1-2+deb10u3 \
-       libc6-dev=2.28-10 \
+    && apt-get install -y --no-install-recommends libgomp1=10.2.1-6 \
+       procps=2:3.3.17-5 \
+       gcc=4:10.2.1-1 \
+       git=1:2.30.2-1 \
+       libc6-dev=2.31-13+deb11u3 \
        libglib2.0-0 \
        libgl1-mesa-glx \
+       libcairo2 \
```
Comment (yeonghoo2) [inline python_with_ffmpeg/Dockerfile:32]: How about using below versions?
[libglib2.0-0=2.66.8-1](https://packages.debian.org/search?keywords=libglib2.0-0)
[libgl1-mesa-glx=20.3.5-1](https://packages.debian.org/search?keywords=libgl1-mesa-glx)
[libcairo2=1.16.0-5](https://packages.debian.org/search?keywords=libcairo2)

**Review by yeonghoo2 — COMMENTED** [review]
reviewed

File: python_with_ffmpeg/Dockerfile (hunk 1 above)
Comment (yeonghoo2) [reply python_with_ffmpeg/Dockerfile:32]: and can you add `gitignore` file? 😃

Comment (nalssing) [reply python_with_ffmpeg/Dockerfile:32]: I'm not sure why their version is fixed. Actually I think we can delete all version constraints

Comment (yeonghoo2) [reply python_with_ffmpeg/Dockerfile:32]: I thought fixing the version is more stable than before like not using latest docker image.
and It's just a suggestion if we need it. 👍

**Review by yeonghoo2 — APPROVED** [review]
approved.

**Pull request merged**